trial_mq/README.md

```diff
@@ -0,0 +1,23 @@
+Trial IronMQ with Terraform
+===========================
+
+If you have an Amazon Web Services account, you can try IronMQ locally with this Terraform script.
```

Comment:
"you can automatically provision an IronMQ cluster there using these Terraform scripts."